.gitignore

```diff
@@ -15,3 +15,4 @@ yarn-error.log
 
 # sass build artifacts
 src/**/*.css
+/sample-app/src/css-build
```

Comment:
is this for WC?

Reply:
yeah, I updated the repo to get the sample app working again so that the WC team could pull this branch and see the behavior for themselves.

Reply:
got it, sweet 👍🏼